src/add-api/index.md

```diff
@@ -0,0 +1,31 @@
+# How do I add scoreboards and trophies to my game?
+
+First, you need to implement the Game API in your game's code. Then, you can add scoreboards and trophies to your game.
+
+## Scoreboards
+
+You can add multiple scoreboards to a game, so you could have, for example, one for each level of the game, as well as a global scoreboard. Or, you could make one that ranks players by something besides points, like total time or number of jumps. You have control over sorting options, so you could make a traditional scoreboard, where the same person can occupy multiple slots, or you could make it so that only someone's best score is displayed. You can enable guest scoring, so players don't need an account to submit a score. You can even attach extra, hidden data to scores (for verification, tracking, or other uses).
+
+## Trophies
+
+Trophies come in bronze, silver, gold, and platinum; the material corresponds to how easy or difficult they are to earn. You can give your trophies custom art and clever titles. You can make secret trophies that aren't completely revealed until they are won. You can keep trophies hidden to test them out.
+
+### What are the requirements and guidelines for trophy images?
+
+WAITING ON INFO
```

Comment:
metnion jpg, gif, png. png is best

max filesize is 3mb. max dimensions is 2000x2000px. It's a square image. We will put black borders around the image to make sure it's square.